driftbase/api/matches.py

```diff
@@ -674,6 +680,37 @@ def get(self, match_id, player_id):
         ret["player_url"] = url_player(player_id)
         return jsonify(ret)
 
+    @requires_roles("service")
+    @bp.arguments(MatchPlayerPutRequestSchema)
+    def put(self, args, match_id, player_id):
```

Comment:
Updating a resource should use "patch", not "put". Those places which use "put" are legacy due to early clients not supporting "patch"